### ISSUE TITLE
License header does not match /usr/portage/header.txt

### ISSUE THREAD

**Issue by PMaynard** [gentoo/gentoo-syntax#23]
Hello, 

The contents of my /usr/portage/header.txt show: 

```
# Copyright 1999-2019 Gentoo Authors
# Distributed under the terms of the GNU General Public License v2
```

However, when I create an empty .ebuild the header looks like this: 
```
# Copyright 2019 Gentoo Authors
# Distributed under the terms of the GNU General Public License v2
```

Wondering if the plugin should match header.txt, or the other way round? :) 
I can across this through the wiki on writing Ebuild.
https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds#Example:_Package_for_a_file_that_echoes_.22hello_world.21.22

**Comment by floppym** [comment]
I think the current behavior of gentoo-syntax is more correct; claiming copyright for a file that did not exist in 1999 is weird.

Gentoo QA policy permits either form.

**Comment by ulm** [comment]
> I think the current behavior of gentoo-syntax is more correct; claiming copyright for a file that did not exist in 1999 is weird.

+1 for only the current year.

When we drafted GLEP 76, I had discussed the format of the copyright notice with @silverhook (who is a lawyer). The conclusion was that the years aren't that important, and IIRC the last year is more important than the first.

**Comment by silverhook** [comment]
For reference, since I encountered several questions regarding copyright statements in the wild, I assembled them in one blog post:

https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code

TL;DR: IMHO the easiest and most correct year is the one of the file’s creation. And then just don’t touch the year any more.
